pytest/tests/sanity/proxy_example.py

```diff
@@ -20,12 +20,13 @@
 from multiprocessing import Value
 
 TIMEOUT = 30
-success = Value('i', 0)
 
 
 class Handler(ProxyHandler):
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, success=None, **kwargs):
+        assert success is not None
+        self.success = success
```

Comment:
I see this logic repeated, could it go to ProxyHandler?

Reply:
Some of them don't use the flag. The main issue however is that if it goes into the ProxyHandler there's no easy way for the parent process (main code) to access it.